codec/src/main/java/io/netty/handler/codec/CharSequenceValueConverter.java

```diff
@@ -66,8 +66,15 @@ public CharSequence convertFloat(float value) {
 
     @Override
     public boolean convertToBoolean(CharSequence value) {
+        if (value instanceof String) {
```

Comment:
This case doesn't seem to be all that necessary. Not a big deal though.

Reply:
yep we can just remove it I think..